projects/public-search/src/app/document-detail/request/pickup-location/pickup-location.component.html

```diff
@@ -0,0 +1,45 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2021 RERO
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<ng-container *ngIf="fields.length > 0 && showForm">
+  <div id="pickup-location-{{ item.metadata.pid }}" class="row pl-4 col-12">
+    <dt class="mb-0 col-lg-2 col-sm-3">&nbsp;</dt>
+    <dd class="mb-0 col-lg-10 col-sm-9">
+      <form class="my-2" [formGroup]="form" (ngSubmit)="submit()">
+        <formly-form [model]="model" [fields]="fields" [form]="form"></formly-form>
+        <button type="submit" class="btn btn-sm btn-primary" [disabled]="!form.valid || requestInProgress">
+          <ng-container *ngIf="!requestInProgress; else inProcess">
+            {{ 'Valid' | translate }}
```

Comment:
'Valid your request' ? (See with PO)

Reply:
The verb is *Validate*.

But it it's the button to confirm the request after selecting the pickup location, then it may be something like *Confirm you request* or *Apply the request*.

Reply:
done